wolfcrypt/src/asn.c

```diff
@@ -41380,8 +41428,14 @@ int ParseX509Acert(DecodedAcert* acert, int verify)
 
         /* Determine which issuer tag was seen. We need this to determine
          * the holder_input. */
-        i_issuer = (dataASN[ACERT_IDX_ACINFO_CHOICE_SEQ].tag != 0) ?
-                    ACERT_IDX_ACINFO_CHOICE_SEQ : ACERT_IDX_ACINFO_ISSUER_SEQ;
+        i_issuer = (dataASN[ACERT_IDX_ACINFO_ISSUER_V2].tag != 0) ?
+                    ACERT_IDX_ACINFO_ISSUER_V2 : ACERT_IDX_ACINFO_ISSUER_V1;
+
+        #ifdef WOLFSSL_DEBUG_ASN_TEMPLATE
```

Comment:
Did you intend to leave these?

Reply:
I would prefer to keep these if ok, because some of the example acerts have non-standard holder or issuer fields.